mithril-aggregator/src/services/snapshotter/compressed_archive_snapshotter.rs

```diff
@@ -632,7 +682,8 @@ mod tests {
             let test_dir = temp_dir_create!();
             let cardano_db = DummyCardanoDbBuilder::new(current_function!())
                 .with_immutables(&[1, 2, 3])
-                .with_ledger_files(&["437", "537", "637", "737", "9not_included"])
+                .with_legacy_ledger_snapshots(&[437, 537, 637, 737])
+                .with_non_ledger_files(&["9not_included"])
```

Comment:
There are cases where the legacy ledger snapshots filename could be different than an integer.
Maybe the previous way of handling them was a better choice.

Reply:
Future ledger state snapshots created to be certified by a multi-signature will be created with a `-mithril` suffix.
I'm not convinced that introducing the `with_non_ledger_files` is the best choice, maybe keeping the `with_legacy_ledger_snapshots(&[&str])` signature is better.